lib/auth/AuthInfo.js

```diff
@@ -49,6 +49,14 @@ class AuthInfo {
     isRequesterPublicUser() {
         return this.canonicalID === constants.publicId;
     }
+    isRequesterAServiceAccount() {
+        return this.canonicalID.startsWith(
+            `${constants.zenkoServiceAccountsNs}/`);
```

Comment:
Ns for namespace? How about zenkoServiceAccountId since we use publicId?

Reply:
I prefer to name it a namespace (or prefix), because it's not an account id but the prefix at which all service accounts will go, the full account id must have the service name appended to the prefix. As for publicId it's a complete account ID.